app/docs/Auth/LoginController.js

```diff
@@ -0,0 +1,34 @@
+/**
+ *  @api {post} /auth/login Attempt to log
+ *  @apiName PostAuthLogin
+ *  @apiGroup Auth
+ *  @apiVersion 1.0.0
+ *
+ *  @apiParam {String} email Users email
+ *  @apiParam {String} password Users password
+ *
+ *  @apiParamExample {json} Request-Example:
+ *     {
+ *       "authToken": ""
```

Comment:
please format it, sometimes it is 2-spaces nested, sometimes 4-spaces, in all files